tests/python/unittest/test_symbol.py

```diff
@@ -89,7 +89,7 @@ def test_symbol_infer_shape():
     x2h  = mx.symbol.FullyConnected(data=data, name='x2h', num_hidden=num_hidden)
     h2h  = mx.symbol.FullyConnected(data=prev, name='h2h', num_hidden=num_hidden)
 
-    out  = mx.symbol.Activation(data=x2h+h2h, name='out', act_type='relu')
+    out  = mx.symbol.Activation(data=mx.sym.ewise_plus(x2h, h2h), name='out', act_type='relu')
```

Comment:
If you dont change x2h+h2h to mx.sym.ewise_plus(x2h, h2h) will this break?
This is a major compatibility break